apps/chat/src/components/Chat/Publish/ReviewApplicationDialogView.tsx

```diff
@@ -103,10 +103,10 @@ export function ReviewApplicationDialogView() {
               </div>
             </div>
           )}
-        {application?.maxInputAttachments !== undefined && (
+        {application?.maxInputAttachments && (
           <div className="flex gap-4">
             <span className="w-[122px] text-secondary">
-              {t('Max. attachments number:')}
+              {t(' Max. attachments number:')}
```

Comment:
```suggestion
              {t('Max. attachments number:')}
```

Reply:
there is another problem, if I don't use space, the text doesn't show, you can check

Reply:
when there is a "." symbol in a line, the line is not displayed, probably the problem is due to the library configuration